# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/linux24/82-raid5-niceer-output.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


I also sent this patch to the kernel hackers - because the output is
really confusing when the software RAID code does not choose the fastest
mode ... Rene

--- linuxi.orig/drivers/md/xor.c	Sun Sep 30 21:26:06 2001
+++ linux/drivers/md/xor.c	Fri Dec  7 18:16:10 2001
@@ -127,6 +127,8 @@
 
 #ifdef XOR_SELECT_TEMPLATE
 	fastest = XOR_SELECT_TEMPLATE(fastest);
+	printk("raid5: using %s to write around L2 cache\n",
+		fastest->name);
 #endif
 
 	active_template = fastest;
